source/extensions/clusters/eds/eds.h

```diff
@@ -61,6 +64,12 @@ class EdsClusterImpl
     return !name.empty() ? name : info_->name();
   }
 
+  // Updates the internal data-structures with a given cluster load assignment.
```

Comment:
nit: dash not required in data-structures

Reply:
Done.